src/R/Components/Impl/ConnectionManager/Implementation/ViewModel/ConnectionViewModel.cs

```diff
@@ -100,6 +101,29 @@ internal sealed class ConnectionViewModel : BindableBase, IConnectionViewModel {
             set { SetProperty(ref _isConnected, value); }
         }
 
+        /// <summary>
+        /// Tooltip when hovered over connection name
+        /// </summary>
+        public string Information {
```

Comment:
OK. I think I might rename it to Tooltip since the purpose is formatted content. If we want to display information elsewhere, it will probably be built differently, not as a string. And if this is just for tooltip then it doesnt need changed event.